ColabTool/Shared/PostFooter/index.jsx

```diff
@@ -57,9 +58,20 @@ const renderEdit = ({ onEditClick }) =>
   </span>;
 
 const renderApproval = ({
+  isPastDue,
   onApproveClick,
+  onEditClick,
   manager,
 }) => {
+  if (isPastDue) {
+    return (<span className={style['post-button-last']}>
+      <span className={style['vertical-line']} />
+      <Button onClick={onEditClick} noStyle>
```

Comment:
Heya @hharnisc 😄 Good one!

So currently the 'Reschedule' option would just open the Edit composer with the calendar already open. 🤔 Thinking about it now, though - it does seem like it's a separate event. 

Feels great to  change this 👍  I'll go ahead and add an `onRescheduleClick` 😄 Thanks!